features/facebookImport/src/views/ministories/story.jsx

```diff
@@ -1,10 +1,11 @@
 import { Component } from "react";
 
 class Story extends Component {
-    constructor({ analyses }) {
+    constructor({ account, mode }) {
         super();
-        this._analyses = analyses;
+        this._analyses = account.analyses;
         this._neededAnalyses = [];
+        this._mode = Story.MODES[mode] || Story.MODES.SUMMARY;
```

Comment:
Effectively, this makes `SUMMARY` the default, right? Maybe just move it to the arguments.

Reply:
I guess that the fact that `render` is called on the components makes `mode` a state. Still, we should maybe think if this is what we want. I would assume that `summary` is the default, and if the state is changed to other, after rendering, it should flip to `summary` again, right?

Reply:
I was hoping `mode` could be a prop - now it's not even React state, but JS state. Do we actually have situations where the JS object for the component is not being recreated, but it's being rerendered?

Reply:
I thought this is what you guys wanted from the mode. It's easy to change it not to be state